scripts/check_clean_enlistment.sh

```diff
@@ -12,11 +12,9 @@
 #
 # Examples: scripts/check_clean_enlistment.sh
 
-# Treat travis builds as 'clean'
-if [ -z "${TRAVIS_BRANCH}" ]; then
-    if [[ -n $(git status --porcelain) ]]; then
-        exit 1
-    fi
+# Check for clean enlistment ignoring the update to buildnumber.dat which is expected during a release build
+if [ $(git status --porcelain | grep -v "buildnumber.dat" | wc -l) == 0 ]; then
```

Comment:
I don't think that this would work. You need to have a single "=" rather than two.

Reply:
== works, however, I replaced the enclosing brackets with double parens so that it is a numeric comparison rather than a string comparison.

Reply:
yes, that's a good idea as on Mac `wc -l` is adding spaces.